resources/logging/values.yaml

```diff
@@ -67,11 +70,13 @@ persistence:
   size: 200Gi
```

Comment:
Does this gets applied to cluster only? I don't see a flag to control it.

Reply:
Line 46 is for local environments 
https://github.com/kyma-project/kyma/blob/fe22e90700f90d46564dc1bdeff9abc7c63a9923/resources/logging/values.yaml#L46
And line 70 is cluster https://github.com/kyma-project/kyma/blob/fe22e90700f90d46564dc1bdeff9abc7c63a9923/resources/logging/values.yaml#L67 

Are managed by, https://github.com/kyma-project/kyma/blob/80d7bc75611a1b7398c04bfa87519ef0f19dba18/installation/resources/installer-config-cluster.yaml.tpl and https://github.com/kyma-project/kyma/blob/1e8ef70ee907054ead854df793b57d22c896861c/installation/resources/installer-config-local.yaml.tpl

Reply:
Line 46 is for local environments 
https://github.com/kyma-project/kyma/blob/fe22e90700f90d46564dc1bdeff9abc7c63a9923/resources/logging/values.yaml#L46
And line 70 is cluster https://github.com/kyma-project/kyma/blob/fe22e90700f90d46564dc1bdeff9abc7c63a9923/resources/logging/values.yaml#L67 

Are managed by, https://github.com/kyma-project/kyma/blob/80d7bc75611a1b7398c04bfa87519ef0f19dba18/installation/resources/installer-config-cluster.yaml.tpl and https://github.com/kyma-project/kyma/blob/1e8ef70ee907054ead854df793b57d22c896861c/installation/resources/installer-config-local.yaml.tpl